recipes/etl/all/conanfile.py

```diff
@@ -12,7 +12,6 @@ class EmbeddedTemplateLibraryConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     description = "A C++ template library for embedded applications"
     topics = ("c++", "embedded", "template", "container", "utility", "framework", "messaging")
-    settings = "os", "arch", "compiler", "build_type"
```

Comment:
it should be kept. 2.0 will complain about missing settings.

Reply:
I see.
reverted